### PR TITLE
Create 1.28 foundation

### PR DESCRIPTION
**Pull request by bhavi-koduru**
*Issue #, if available:*
https://github.com/aws/eks-distro/issues/2206

*Description of changes:*
Everything is copied from 1.27

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/kubernetes/release/1-28/images/build/go-runner/go.mod
Comment (markapruett) [inline projects/kubernetes/release/1-28/images/build/go-runner/go.mod:1]: Why is this blank?

Comment (kschumy) [reply projects/kubernetes/release/1-28/images/build/go-runner/go.mod:1]: It’s blank in all versions. https://github.com/aws/eks-distro/blob/main/projects/kubernetes/release/1-23/images/build/go-runner/go.sum

**Pull request merged**